lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
Ah, sorry for booleans (and possibly selects?), if the (true|false)value is an empty string, then it will display `Not available.`, e.g. from kraken:
```
<param name="only_classified_output" type="boolean" checked="False" truevalue="--only-classified-output" falsevalue="" label="Print no Kraken output for unclassified sequences" help="--only-classified-output"/>
```
results in:
![image](https://cloud.githubusercontent.com/assets/1231190/19365562/53e386bc-9161-11e6-95d1-22926d026b3a.png)
when it is False.